changelog.d/21822-nats-sink-multiple-urls.md

```diff
@@ -0,0 +1,3 @@
+Add possibility to provide multiple urls in NATS sink.
```

Comment:
The `nats` sink can now parse comma separated urls.

Reply:
Fixed the changelog